### ISSUE TITLE
Fix dottydoc build for support modules

### ISSUE THREAD

**Issue by larsrh** [typelevel/jawn#271]
I tried releasing a new Jawn version cross-built against Scala 2.x and Dotty. Unfortunately, the doc build fails for the support modules. (For now, I've back-released Jawn 1.0.0 without the support modules.)

So far this wasn't caught because publishing the doc artifacts is disabled for Dotty:

```diff
diff --git a/build.sbt b/build.sbt
index dd17cf1..85cb736 100644
--- a/build.sbt
+++ b/build.sbt
@@ -66,7 +66,6 @@ lazy val jawnSettings = Seq(
   releaseVcsSign := true,
   publishMavenStyle := true,
   publishArtifact in Test := false,
-  publishArtifact in (Compile, packageDoc) := !isDotty.value,
   pomIncludeRepository := Function.const(false),
   publishTo := {
     val nexus = "https://oss.sonatype.org/"
```

Ideally, we should figure this out before the next release.

**Comment by larsrh** [comment]
@djspiewak suggests the following alternative workaround:

```scala
Compile / doc / sources ~= { old => if (isDotty.value) Seq() else old }
```

**Comment by larsrh** [comment]
Fixed in 02e71c99876e9db55b40a088b2345b111e1353fa
